.github/workflows/test.yml

```diff
@@ -39,7 +42,7 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
       - name: execute test run
-        run: make test-unit-race
+        run: make SKIP_CI=${{ env.SKIP_CI }} test-unit-race
```

Comment:
my only concern with this approach is that we'll likely never run these tests other than in CI 🤔

Reply:
There is risk. But it's manageable for now if we don't plan to pump more test cases with core 🤞